<launch>
  <node pkg="multi_lidar_calibration" type="multi_lidar_calibration_node" name="multi_lidar_calibration_node" output="screen">
    <param name="use_sim_time" value="true" />
    <rosparam file="$(find multi_lidar_calibration)/config/params.yaml" command="load"/>
  </node>

  <!--- Run Rviz-->
      <!--- Run Rviz-->
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find multi_lidar_calibration)/rviz/multi_lidar_calibration.rviz" />

  <!-- 播放 rosbag 文件 -->
  <node pkg="rosbag" type="play" name="rosbag_play" args="--loop --rate 1.0 --clock $(find multi_lidar_calibration)/data/2024-08-02-11-03-54.bag">
  <param name="use_sim_time" value="true" />
  </node>
</launch>
